index.js

```diff
@@ -331,7 +330,10 @@ var Cardboard = module.exports = function(config) {
         }
 
         pageOptions = pageOptions || {};
-        if (pageOptions.start) opts.start = pageOptions.start;
+        if (pageOptions.start) opts.start = {
```

Comment:
If you roll this with https://github.com/mapbox/dyno/pull/104 the wire-formatting here would go away, right? That would